### PR TITLE
added utility to update target server and update kvm entry from sdk a…

### PR DESCRIPTION
**Pull request by riteshverma87**
…nd command line

### CODE REVIEW

File: lib/commands/updateKVMentry.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    prompt: true
+  }
+
+});
+
+module.exports.descriptor = descriptor;
+
+module.exports.run = function (opts, cb) {
+  if (opts.debug) {
+    console.log('getKVMentry: %j', opts);
```
Comment (OmidTahouri) [inline lib/commands/updateKVMentry.js]: Fix value. This isn't `getKVMEntry`

File: lib/main.js [hunk 2]
```diff
@@ -229,8 +229,21 @@ ApigeeTool.listRoleUsers = function (opts, cb) {
   runCommand(cmd, opts, cb);
 };
 
+ApigeeTool.updateTargetServer = function (opts, cb) {
```
Comment (OmidTahouri) [inline lib/main.js]: Please move each new function closer to its friends. i.e. KVM functions together, Target Server functions together

File: lib/promisesdk.js [hunk 3]
```diff
@@ -12,330 +12,342 @@ function ApigeeTool(defaults) {
 
 module.exports = ApigeeTool;
 
-ApigeeTool.defaults = function(newDefaults) {
-  var tool= new ApigeeTool(newDefaults);
+ApigeeTool.defaults = function (newDefaults) {
```
Comment (OmidTahouri) [inline lib/promisesdk.js]: Please undo the whitespace changes throughout this file in order to achieve a leaner and cleaner commit/diff.

File: lib/promisesdk.js [hunk 4]
```diff
@@ -12,330 +12,342 @@ function ApigeeTool(defaults) {
...
   return cb.promise
 }
-ApigeeTool.listRoleUsers = function(opts) {
+ApigeeTool.listRoleUsers = function (opts) {
   var cb = q.defer()
   var cmd = require('./commands/listRoleUsers');
   runCommand(cmd, opts, cb);
   return cb.promise
 }
+ApigeeTool.updateTargetServer = function (opts) {
```
Comment (OmidTahouri) [inline lib/promisesdk.js]: As before, please move each function closer to its friends (where functions for other operations on same entity are)

File: lib/main.js [hunk 5]
```diff
@@ -229,8 +229,21 @@ ApigeeTool.listRoleUsers = function (opts, cb) {
...
+
+ApigeeTool.updateKVMentry = function (opts, cb) {
+  var cmd = require('./commands/updateKVMentry');
+  runCommand(cmd, opts, cb);
+};
+
+
 function runCommand(cmd, opts, cb) {
-  options.validate(opts, cmd.descriptor, function(err) {
+  console.log('test')
```
[the remark this answers is not in the document]
Comment (OmidTahouri) [reply lib/main.js]: Resolved

**Comment by OmidTahouri** [comment]
Please make appropriate updates to the README.md

**Comment by OmidTahouri** [comment]
@riteshverma87 @ritzverma -- it seems you have two identities, with one missing a CLA. Please see [this comment from @googlebot](https://github.com/apigee/apigeetool-node/pull/183#issuecomment-578809505) and follow the instructions.

**Comment by riteshverma87** [comment]
@googlebot I fixed it.

**Comment by riteshverma87** [comment]
@googlebot I fixed it.

**Comment by riteshverma87** [comment]
@googlebot I fixed it.

**Comment by riteshverma87** [comment]
@googlebot I consent

**Comment by riteshverma87** [comment]
> @riteshverma87 @ritzverma -- it seems you have two identities, with one missing a CLA. Please see [this comment from @googlebot](https://github.com/apigee/apigeetool-node/pull/183#issuecomment-578809505) and follow the instructions.

Hi Omid,

@ritzverma is my personal github account. I haven't done any commit from that account.I've created the google account using my official email id and signed the CLA using the same. I don't know why it's taking reference of my personal account.

**Comment by OmidTahouri** [comment]
> > @riteshverma87 @ritzverma -- it seems you have two identities, with one missing a CLA. Please see [this comment from @googlebot](https://github.com/apigee/apigeetool-node/pull/183#issuecomment-578809505) and follow the instructions.
> 
> Hi Omid,
> 
> @ritzverma is my personal github account. I haven't done any commit from that account.I've created the google account using my official email id and signed the CLA using the same. I don't know why it's taking reference of my personal account.

@riteshverma87 All of the commits in this PR are authored by @ritzverma

**Comment by riteshverma87** [comment]
@googlebot I fixed it.

**Comment by ritzverma** [comment]
@googlebot I consent.

**Comment by OmidTahouri** [comment]
Thanks, @riteshverma87 
LGTM

**Comment by NasaGeek** [comment]
These new commands don't seem to be accessible from apigeetool. I think they must also be present in ./lib/commands/commands.js

**Pull request merged**